samples/sample_hevc_fei_abr/include/dso/bs_reader2.h

```diff
@@ -184,16 +184,16 @@ class File : public BufferUpdater
 
 #define BS2_TRO
 #define BS2_SET(val, var) { var = (val); }
-#define BS2_TRACE(val, var) {(val);}
-#define BS2_TRACE_STR(str) {(str);}
+#define BS2_TRACE(val, var) {std::ignore = (val);}
```

Comment:
why not?
`#define BS2_TRACE(val, var)`

Reply:
define to empty? should be better, agree